src/Boost.sol

```diff
@@ -1,87 +1,145 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
-import "openzeppelin-contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-contracts/access/Ownable.sol";
 import "openzeppelin-contracts/utils/cryptography/SignatureChecker.sol";
 import "openzeppelin-contracts/utils/cryptography/draft-EIP712.sol";
+
 import "./IBoost.sol";
 
-contract Boost is IBoost, EIP712("boost", "1") {
+contract Boost is IBoost, EIP712("boost", "1"), Ownable {
     bytes32 public immutable eip712ClaimStructHash =
         keccak256("Claim(uint256 boostId,address recipient,uint256 amount)");
 
     uint256 public nextBoostId = 1;
     mapping(uint256 => BoostConfig) public boosts;
     mapping(address => mapping(uint256 => bool)) public claimed;
 
+    // Constant eth fee (in gwei) that is the same for all boost creators.
+    uint256 public ethFee;
+    // The fraction of the total boost deposit that is taken as a fee.
+    // represented as an integer denominator (1/x)%
```

Comment:
it's not a `%` though. If I put 2, I will get 1/x -> 1/2 -> 0.5. If I want a percentage, I need to multiply by 100! :)

Reply:
yep true, ill update the comment. i just copied uniswap: https://github.com/Uniswap/v3-core/blob/05c10bf6d547d6121622ac51c457f93775e1df09/contracts/UniswapV3Pool.sol#L68